### PR TITLE
Fixes unwise else case in Spellbook UI, causing runtimes

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

With specific dextrous use of the spellbook you can both trigger "purchase" and the wizard loadout proc at the same time. Why? Because I am not smart and my backend has more holes in it than swiss cheese

## Why It's Good For The Game

Bugfix

## Changelog

:cl:
fix: Fixes a weird bug with spellbooks that may have been locking some spell options when you did some specific actions
/:cl:

### CODE REVIEW

File: code/modules/antagonists/wizard/equipment/spellbook.dm [hunk 1]
```diff
@@ -795,6 +795,7 @@
 						entry.limit--
 					uses -= entry.cost
 			update_static_data(wizard) //update statics!
+			return
```
Comment (MrMelbert) [inline code/modules/antagonists/wizard/equipment/spellbook.dm:798]: this should return `TRUE` as it needs to update, no? Unless i'm missing something

File: code/modules/antagonists/wizard/equipment/spellbook.dm [hunk 2]
```diff
@@ -804,6 +805,7 @@
 						entry.limit += result
 					uses += result
 			update_static_data(wizard) //update statics!
+			return
```
Comment (MrMelbert) [inline code/modules/antagonists/wizard/equipment/spellbook.dm:808]: ditto

File: code/modules/antagonists/wizard/equipment/spellbook.dm (hunk 1 above)
Comment (Watermelon914) [reply code/modules/antagonists/wizard/equipment/spellbook.dm:798]: yep, looks like the tgui is old and never returned true, probably autoupdates too

**Review by Mothblocks — CHANGES_REQUESTED** [review]
what the uoutuber said

**Pull request merged**